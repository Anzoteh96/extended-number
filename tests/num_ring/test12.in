966239626591529878118712744210922347
724575563968966851110367210359478599
949891611231146491481327193238675188
