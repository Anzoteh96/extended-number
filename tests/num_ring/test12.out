Enter the number base b:
Enter two numbers n and m:
b is 966239626591529878118712744210922347
n mod b is 724575563968966851110367210359478599
m mod b is 949891611231146491481327193238675188
n + m is 708227548608583464472981659387231440
n - m is 740923579329350237747752761331725758
n * m is 791797943779656244748324659049479209
n ** m is 461866131165955025163000887484892300
